clients/admin-ui/src/features/taxonomy/TaxonomyFormBase.tsx

```diff
@@ -146,14 +124,6 @@ const TaxonomyFormBase = ({
                   />
                 ))}
               {renderExtraFormFields ? renderExtraFormFields(values) : null}
-              {["data category", "data use", "data subject"].includes(
-                labels.fides_key.toLowerCase()
-              ) && (
-                <CustomFieldsList
-                  resourceId={initialValues.fides_key}
-                  resourceType={getResourceType(labels.fides_key)}
-                />
-              )}
```

Comment:
This block now lives in `renderExtraFormFields` when appropriate for the resource type. This actually removes the need for any resource type checking.